pegasus/sites.v3/code.org/public/promote/index.haml

```diff
@@ -64,9 +63,10 @@ social:
 
 :javascript
   var petition;
-  var promote;
```

Comment:
Not related to this particular error. It was just some left over js that I cleaned up. Would it have been better to do it separately?